src/App.tsx

```diff
@@ -328,12 +329,16 @@ function App(): ReactElement {
   );
 
   const getFeatureValue = useCallback(
-    (id: number): number => {
+    (id: number): string => {
       if (!featureName || !dataset) {
-        return -1;
+        return "";
       }
       // Look up feature value from id
-      return dataset.getFeatureData(featureName)?.data[id] || -1;
+      const featureData = dataset.getFeatureData(featureName);
+      const featureValue = featureData?.data[id] ?? -1;
```

Comment:
the the data value is 0 will this return -1?

Reply:
Minor/ nit: this whole block of code could actually go inside of dataset.  You already have `dataset.getFeatureNameWithUnits` - you could also have `dataset.getFeatureValueWithUnits(featureName, id)`

Reply:
The `??` is a [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)! It's similar to using `||` but checks only for `null` or `undefined`, so 0 or other falsy values are safe. I'll add a comment about it though!

Reply:
my bad, thanks